pages/jams/[jam].js

```diff
@@ -175,32 +183,41 @@ const Jam = () => {
         description={jam && jam.description}
         participantId={participantId}
       />
+      {question && (
+        <Progress
+          h="0.5"
+          value={
+            (question.meta.numVotes / question.meta.numQuestions) *
+            100
+          }
+        />
+      )}
```

Comment:
The progress bar currently disappears when we complete the Jam (as there's no more `question` object), any way we can show it as complete?

Reply:
We can for sure!